ci/pipeline.yml

```diff
@@ -91,52 +107,96 @@ resources:
 - name: windows2019-stemcell
   type: bosh-io-stemcell
   source:
-    name: bosh-google-kvm-windows2019-go_agent
+    name: bosh-aws-xen-hvm-windows2019-go_agent
```

Comment:
I think the cats env is deployed on GCP, so we would probably want to stick with a google stemcell.

Reply:
Doh - this was left in accidentally when I was stripping out the temporarily changes I had made to get this running on a different foundation on our side. Will revert.